tests/ci/cdk/README.md

```diff
@@ -186,12 +193,12 @@ Below are some commands specific to windows docker image build.
  
 ```bash
 # Define environment variables needed by Windows docker image build.
-export AWS_LC_S3_BUCKET_PREFIX="aws-lc-windows-docker-image-build"
-export DATE_NOW="$(date +%Y-%m-%d-%H-%M)"
+DATE_NOW="$(date +%Y-%m-%d-%H-%M)"
+export AWS_LC_S3_BUCKET_PREFIX='aws-lc-windows-docker-image-build-s3'
 export S3_FOR_WIN_DOCKER_IMG_BUILD="${AWS_LC_S3_BUCKET_PREFIX}-${DATE_NOW}"
-export WIN_EC2_TAG_KEY="aws-lc"
+export WIN_EC2_TAG_KEY='aws-lc'
```

Comment:
Why did we change this to single quotes? Was this being interpreted weirdly?

Reply:
No, the change is to use double quotes on demand.